go/pkg/client/tree_test.go

```diff
@@ -936,6 +942,74 @@ func TestComputeMerkleTree(t *testing.T) {
 				MaterializeOutsideExecRoot: true,
 			},
 		},
+		{
```

Comment:
Can you also add a test with multiple parent symlinks, and another where the file itself is a symlink?

Reply:
I covered that in the unit tests. Covering it here requires a bunch of global variables or a writing a new test with better ergonomics.

Reply:
I guess you mean its covered in the whitebox test? I guess you would need to change the fake tree structuring logic here to support this. sgtm to keep as is.